cgmanifest.json

```diff
@@ -3209,7 +3209,7 @@
         "other": {
           "name": "eglexternalplatform",
           "version": "1.1",
-          "downloadUrl": "https://github.com/NVIDIA/eglexternalplatform/archive/7c8f8e2218e46b1a4aa9538520919747f1184d86.tar.gz"
+          "downloadUrl": "https://github.com/NVIDIA/eglexternalplatform/archive/1.1/eglexternalplatform-1.1.tar.gz"
```

Comment:
I see, but just for clarification, that is the URL given in Fedora's spec file.

https://src.fedoraproject.org/rpms/eglexternalplatform/blob/rawhide/f/eglexternalplatform.spec#_11

and, FYI, that is changed by below.

https://src.fedoraproject.org/rpms/eglexternalplatform/c/9f2aa931aa18d098df78d6985a9d17d2e9905cc4?branch=rawhide

And, source tarball can be downloaded from that URL, so what defines 'real' URL?

Reply:
I changed as you suggested, but now "Check Package CGManifests / Check Package CGManifests" is failing since source name doesn't match with one in spec file. So should I update spec file now? or revert cgmanifest.json?